daffodil-lib/src/main/resources/org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd

```diff
@@ -93,6 +101,17 @@
           utf16Width="fixed"
         />
       </dfdl:defineFormat>
+      
+      <dfdl:defineFormat name="GeneralFormat">
+        <dfdl:format ref="GeneralFormatOriginal" />
+      </dfdl:defineFormat>
+        
+      <dfdl:defineFormat name="GeneralFormatForCrossTesting">
+        <dfdl:format ref="GeneralFormatOriginal"
+          calendarTimeZone="UTC"
+          encodingErrorPolicy="error"
+        />   
+      </dfdl:defineFormat>
```

Comment:
Wondering if this new format should go in src/test/resources somewhere if it really is just for testing purposes? My concen is that anything in this file is probably something we should officially support and might want to worry about backward compatability issues. If it's in src/test/resources, we can changeit whenever we want.

Reply:
Only hassle is then to use it you have to include a different file. There's 500+ include statements to change (eventually). As opposed to 500+ ref="GeneralFormat" statements. One would need to change both. 

Maybe this shouldn't be GeneralFormatForCrossTesting, but GeneralFormatPortable ?

Reply:
Fair. A name change seems like a better solution.